src/GroupTypeManagerInterface.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+namespace Drupal\og;
+
+/**
+ * Interface for services intended to help managing groups.
+ */
+interface GroupTypeManagerInterface {
+
+  /**
+   * Determines whether an entity type ID and bundle ID are group enabled.
+   *
+   * @param string $entity_type_id
+   *   The entity type name.
+   * @param string $bundle
+   *   The bundle name.
+   *
+   * @return bool
+   *   TRUE if a bundle is a group.
+   */
+  public function isGroup($entity_type_id, $bundle);
+
+  /**
+   * Checks if the given entity bundle is group content.
+   *
+   * This is provided as a convenient sister method to ::isGroup(). It is a
+   * simple wrapper for OgGroupAudienceHelperInterface::hasGroupAudienceField().
+   *
+   * @param string $entity_type_id
+   *   The entity type ID.
+   * @param string $bundle
+   *   The bundle name.
+   *
+   * @return bool
+   *   TRUE if the entity bundle is group content.
+   */
+  public function isGroupContent($entity_type_id, $bundle);
+
+  /**
+   * Returns the group of an entity type.
+   *
+   * @param string $entity_type_id
+   *   The entity type name.
+   *
+   * @return \Drupal\Core\Entity\EntityInterface[]
+   *   Array of groups, or an empty array if none found
+   */
+  public function getGroupsForEntityType($entity_type_id);
+
+  /**
+   * Get all group bundles keyed by entity type.
+   *
+   * @return array
+   *   An associative array, keyed by entity type, each value an indexed array
+   *   of bundle IDs.
+   */
+  public function getAllGroupBundles($entity_type = NULL);
+
+  /**
+   * Returns a list of all group content bundles keyed by entity type.
+   *
+   * This will return a simple list of group content bundles. If you need
+   * information about the relations between groups and group content bundles
+   * then use getGroupRelationMap() instead.
+   *
+   * @return array
+   *   An associative array of group content bundle IDs, keyed by entity type
+   *   ID.
+   *
+   * @see \Drupal\og\GroupTypeManager::getGroupRelationMap()
```

Comment:
This `@see` section should be moved back to the GroupTypeManager class, or renamed to match to with the interface.